admin/sql/updates/20220302-mbs-12225-mirror_only.sql

```diff
@@ -0,0 +1,118 @@
+\set ON_ERROR_STOP 1
+
+BEGIN;
+
+ALTER TRIGGER apply_artist_release_group_pending_updates_slave ON release RENAME TO apply_artist_release_group_pending_updates_mirror;
+ALTER TRIGGER apply_artist_release_group_pending_updates_slave ON release_group RENAME TO apply_artist_release_group_pending_updates_mirror;
+ALTER TRIGGER apply_artist_release_group_pending_updates_slave ON release_group_meta RENAME TO apply_artist_release_group_pending_updates_mirror;
+ALTER TRIGGER apply_artist_release_group_pending_updates_slave ON release_group_secondary_type_join RENAME TO apply_artist_release_group_pending_updates_mirror;
+ALTER TRIGGER apply_artist_release_group_pending_updates_slave ON track RENAME TO apply_artist_release_group_pending_updates_mirror;
+ALTER TRIGGER apply_artist_release_pending_updates_slave ON release RENAME TO apply_artist_release_pending_updates_mirror;
+ALTER TRIGGER apply_artist_release_pending_updates_slave ON release_country RENAME TO apply_artist_release_pending_updates_mirror;
+ALTER TRIGGER apply_artist_release_pending_updates_slave ON release_first_release_date RENAME TO apply_artist_release_pending_updates_mirror;
+ALTER TRIGGER apply_artist_release_pending_updates_slave ON release_label RENAME TO apply_artist_release_pending_updates_mirror;
+ALTER TRIGGER apply_artist_release_pending_updates_slave ON track RENAME TO apply_artist_release_pending_updates_mirror;
+
+DROP TRIGGER IF EXISTS a_ins_release_slave ON release;
```

Comment:
You can just rename these instead of dropping and re-creating, because with triggers PG references the functions they call by oid, not name.  Drop & re-create is only really needed for functions that call renamed functions, since the definitions are stored as text.

Reply:
Oh, TIL. Amended.